### ISSUE TITLE
add instructions to readme

### ISSUE THREAD

**Issue by arlin** [phylotastic/phylotastic-portal#31]
I sketched out a rough version of instructions for installing and running the portal app.  Someone else needs to check this and expand it.

**Comment by ducvan0212** [comment]
I added more details in README.md, if there is something that needs to improve, please dont hesitate to modify README.md file

**Comment by arlin** [comment]
Thanks!  I will try to do a clean install on another machine to test the instructions.

**Comment by arlin** [comment]
I'm getting an error.  The nature of the messages suggests that there is a typo in application_controller.rb 

camel14:phylotastic-portal arlin$ bundle exec sidekiq
/Library/Ruby/Gems/2.0.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:457:in `load': /Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:7: syntax error, unexpected ':', expecting => (SyntaxError)
    faker = {"resolvedNames": []}
                             ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:9: syntax error, unexpected ':', expecting =>
      faker[:resolvedNames] << { "match_type": "Exact", 
                                              ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:9: syntax error, unexpected ',', expecting keyword_end
      faker[:resolvedNames] << { "match_type": "Exact", 
                                                       ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:10: syntax error, unexpected ':', expecting keyword_end
                                  "resolver_name": "OT", 
                                                  ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:10: syntax error, unexpected ',', expecting keyword_end
                                  "resolver_name": "OT", 
                                                        ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:11: syntax error, unexpected ':', expecting keyword_end
                                  "matched_name": n, 
                                                 ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:12: syntax error, unexpected ':', expecting :: or '[' or '.'
                                  "search_string": n, 
                                                  ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:13: syntax error, unexpected ':', expecting :: or '[' or '.'
                                  "synonyms": [] }
                                             ^
/Users/arlin/github/local/phylotastic-portal/app/controllers/application_controller.rb:24: syntax error, unexpected end-of-input, expecting keyword_end
	from /Library/Ruby/Gems/2.0.0/gems/activesupport-4.2.3/lib/active_support/dependencies.rb:457:in `block in load_file'

**Comment by ducvan0212** [comment]
I fixed this error, Professor.

**Comment by arlin** [comment]
instructions updated to include how to create db and user, edit config file.  I took out the detailed ruby installation instructions because these were ubuntu-specific and were copied from another site anyway.  I just kept the URL for that.

**Comment by arlin** [comment]
I'm going to close this and create a new ticket to write and test installation instructions for the portal.  This will be required before we can write a publication about the portal.
